packages/datagrid/src/components/DataGrid/DataGrid.component.js

```diff
@@ -330,7 +330,11 @@ export default class DataGrid extends React.Component {
 	render() {
 		let content;
 		if (this.props.loading) {
-			content = <Skeleton name="talend-table" type={Skeleton.TYPES.icon} />;
+			content = (
+				<div className={theme['td-grid-loader']}>
+					<Icon name="talend-table" />
+				</div>
```

Comment:
Do we no longer have a shorter-hand version of this with the design system ?

Reply:
Nope. And the notion of a skeleton icon is _weird_. In the future we'll hopefully provide loaders.